lib/functional.fnl

```diff
@@ -26,7 +29,7 @@
 
 (fn has-some?
   [list]
-  (and list (> (length list) 0)))
+  (and list (< 0 (length list))))
```

Comment:
No objection here but what makes this order better to you?

Reply:
It's a convention, LISPers almost always use `<` and try to avoid using `>` so it's less confusing. It might get confusing with a prefix notation and more than just two items.

Reply:
Ah cool, I did not know that. Thanks!